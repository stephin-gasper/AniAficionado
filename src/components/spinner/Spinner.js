import React from 'react';
import {Image} from 'react-native';
import LoaderGif from '../../assets/gifs/dojutsu.gif';
import {Container, styles} from './Spinner.style';

const Spinner = () => (
  <Container>
    <Image style={styles.spinner} source={LoaderGif} />
  </Container>
);

export default Spinner;
