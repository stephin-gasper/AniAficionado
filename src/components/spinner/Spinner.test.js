import React from 'react';
import {render} from 'test/test-utils';

import Spinner from './Spinner';

describe('<Spinner />', () => {
  it('should render correctly', () => {
    expect(render(<Spinner />)).toMatchSnapshot();
  });
});
