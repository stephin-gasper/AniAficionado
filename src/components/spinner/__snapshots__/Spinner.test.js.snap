// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should render correctly 1`] = `
<View
  style={
    [
      {
        "alignItems": "center",
        "backgroundColor": "rgba(106,84,153,0.7)",
        "bottom": 0,
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 1,
      },
    ]
  }
>
  <Image
    source={
      {
        "testUri": "../../../src/assets/gifs/dojutsu.gif",
      }
    }
    style={
      {
        "height": 140,
        "width": 140,
      }
    }
  />
</View>
`;
